waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/StaticDatabaseMappingService.java

```diff
@@ -360,28 +361,29 @@ public PanopticOperationHandler getPanopticOperationHandler() {
 
       @Override
       public List<TableMeta> getTableMeta(String db_patterns, String tbl_patterns, List<String> tbl_types) {
+        String internal_pattern = FederatedHMSHandler.getDbInternalName(db_patterns);
```

Comment:
```suggestion
        String internalPattern = FederatedHMSHandler.getDbInternalName(db_patterns);
```
(we use camelCase for everything except what has been auto-generated from implementing Hive's interfaces which you see in the method arguments here).